tenable/base.py

```diff
@@ -338,9 +339,14 @@ class APISession(object):
     Retry-After header was returned.
     '''
 
+    _timeout = 120
```

Comment:
With an unset timeout, pyTenable would connect to the server, but if the read request was not responded the scripts would hang indefinitely - necessitating killing the pid and restarting the script (oftentimes days if unnoticed).  Is making it longer more acceptable rather than leaving as none?

I was referencing the following note from requests when electing the 120 seconds, as it pertained to the response and no bytes being received:

> timeout is not a time limit on the entire response download; rather, an exception is raised if the server has not issued a response for timeout seconds (more precisely, if no bytes have been received on the underlying socket for timeout seconds). If no timeout is specified explicitly, requests do not time out.

Reply:
I see the benefit of forcibly resetting the connection if the connection is taking too long (hence why I had accepted the PR with the condition above) however as the current expected behavior is to _not_ timeout, this would effectively introduce breakage into existing code that may be relying on no timeout being set.  exposing the parameter and making it an optional  settings allows for the greatest compromise.  Both allowing for the parameter to be set when desired, and also allowing for existing code to continue to operate as expected.